### DISCUSSION TITLE
search gene method

### DISCUSSION THREAD

**Discussion by Mr-qiang** [GMOD/jbrowse-components#3828] [Q&A]
In jbrowse1, you can use the url to query the result of gene location information. How to implement this function in jbrowse2?
![image](https://github.com/GMOD/jbrowse-components/assets/42408207/6a982ebc-c7f4-4c3d-9313-3fd5ea9293ad)

**Comment by rbuels** [comment]
Have a look at the URL query parameter API, this is the equivalent functionality for JBrowse 2: https://jbrowse.org/jb2/docs/urlparams/
